From c96bb6f9204830f5972021d01b1207fed1b77aa3 Mon Sep 17 00:00:00 2001
From: Akash Gajjar <Akash_Gajjar@mentor.com>
Date: Wed, 13 Feb 2019 22:36:38 +0530
Subject: arm64: dts: rockchip: rockpro64 dts remove unused lcd-reset pinmux

lcd panel pinmux is unused and the pin actually for something different,
so removing it.

Signed-off-by: Akash Gajjar <Akash_Gajjar@mentor.com>
Signed-off-by: Heiko Stuebner <heiko@sntech.de>
---
 arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts | 6 ------
 1 file changed, 6 deletions(-)

--- a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
@@ -523,12 +523,6 @@
 		};
 	};
 
-	lcd-panel {
-		lcd_panel_reset: lcd-panel-reset {
-			rockchip,pins = <4 RK_PD6 RK_FUNC_GPIO &pcfg_pull_up>;
-		};
-	};
-
 	pcie {
 		pcie_pwr_en: pcie-pwr-en {
 			rockchip,pins = <1 RK_PC1 RK_FUNC_GPIO &pcfg_pull_none>;
